mnefun/_mnefun.py

```diff
@@ -842,6 +897,93 @@ def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
     run_subprocess(cmd)
 
 
+def run_sss_localy(p, subjects, run_indices):
+    """Run SSS locally using maxwell filter in python
+
+    .. warning:: Automatic bad channel detection is not currently implemented.
+                 It is critical to mark bad channels before running Maxwell
+                 filtering, so data should be inspected and marked accordingly
+                 prior to running this algorithm.
+
+    .. warning:: Not all features of Elekta MaxFilter™ are currently
+                 implemented (see Notes). Maxwell filtering in mne-python
+                 is not designed for clinical use.
+
+    Notes
+    -----
+    Compared to Elekta's MaxFilter™ software, Maxwwell filtering in mne-python
+    does not require/support the following arguments:
+        -format
+        -hpicons
+        -autobad
+        -force
+
+    """
+    cal_file = op.join(op.dirname(op.dirname(__file__)), 'mnefun/data/sss_cal.dat')
```

Comment:
you can get rid of one of the `op.dirname` calls and also get rid of `mnefun`. Also don't use `/` use `op.join` properly